python-interface/src/MiniBotFramework/Lib/minibot_tools.py

```diff
@@ -0,0 +1,16 @@
+def distance(p1, p2):
```

Comment:
This file should go in a Util folder; Lib is just for things not written by us that we really should not modify.